src/styles/components/Header.scss

```diff
@@ -66,10 +66,68 @@
         line-height: 1.5rem;
         padding-top: 12px;
     }
-    
+
     &__nav-item a {
         margin: 0 10px;
         text-decoration: none;
         color: $color-grey-text;
     }
-}    
+}
+
+.hero {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 16px;
+    width: 100%;
+    height: auto;
+    overflow-x: scroll;
+   overscroll-behavior-x: contain;
+    scroll-snap-type: x proximity;
+    background-color: $color-bc-hero;
+
+    &__component {
+        scroll-snap-align: center;
+        display: flex;
+        width: 246px;
+        height: 128px;
+        margin-right: 16px;
+        padding: 16px;
+        box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+        background: linear-gradient(105.15deg, #5465FF 0%, #B654EF 100%);
+
+        &--description {
+            font-family: 'Josefin Sans', sans-serif;
+            font-size: 1.4rem;
+            font-weight: 700;
+            line-height: 1.8rem;
+            color: $color-white;
+
+
+            &-title {
+                font-family: 'Josefin Sans', sans-serif;
+                font-size: 1.4rem;
+                font-weight: 700;
+                line-height: 1.8rem;
+                color: $color-white;
+            }
+            &-link {
+                font-family: 'Lato', sans-serif;
```

Comment:
same here, move font-family to _vars.scss and use variable instead eg:

`font-family: $font-lato;`